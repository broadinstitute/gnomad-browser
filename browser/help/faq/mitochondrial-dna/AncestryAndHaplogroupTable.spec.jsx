import renderer from 'react-test-renderer'
import { renderAnswer } from './how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population'

test('has no unexpected changes', () => {
  const tree = renderer.create(renderAnswer())
  expect(tree).toMatchSnapshot()
})
