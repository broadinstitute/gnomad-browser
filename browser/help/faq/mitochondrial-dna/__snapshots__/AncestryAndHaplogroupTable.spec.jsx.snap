// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has no unexpected changes 1`] = `
Array [
  <select
    className="Select-sc-1lkyg9e-0 how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population__HaploAncestrySelect-dzc7cx-1 jTBNjj"
    id="haplogroup-selected"
    onChange={[Function]}
    value="All"
  >
    <optgroup
      label="Haplogroup"
    >
      <option
        value="All"
      >
        All Haplogroups
      </option>
      <option
        value="A"
      >
        A
      </option>
      <option
        value="B"
      >
        B
      </option>
      <option
        value="C"
      >
        C
      </option>
      <option
        value="D"
      >
        D
      </option>
      <option
        value="E"
      >
        E
      </option>
      <option
        value="F"
      >
        F
      </option>
      <option
        value="G"
      >
        G
      </option>
      <option
        value="H"
      >
        H
      </option>
      <option
        value="HV"
      >
        HV
      </option>
      <option
        value="I"
      >
        I
      </option>
      <option
        value="J"
      >
        J
      </option>
      <option
        value="K"
      >
        K
      </option>
      <option
        value="L0"
      >
        L0
      </option>
      <option
        value="L1"
      >
        L1
      </option>
      <option
        value="L2"
      >
        L2
      </option>
      <option
        value="L3"
      >
        L3
      </option>
      <option
        value="L4"
      >
        L4
      </option>
      <option
        value="L5"
      >
        L5
      </option>
      <option
        value="M"
      >
        M
      </option>
      <option
        value="N"
      >
        N
      </option>
      <option
        value="P"
      >
        P
      </option>
      <option
        value="R"
      >
        R
      </option>
      <option
        value="T"
      >
        T
      </option>
      <option
        value="U"
      >
        U
      </option>
      <option
        value="V"
      >
        V
      </option>
      <option
        value="W"
      >
        W
      </option>
      <option
        value="X"
      >
        X
      </option>
      <option
        value="Y"
      >
        Y
      </option>
      <option
        value="Z"
      >
        Z
      </option>
    </optgroup>
  </select>,
  <select
    className="Select-sc-1lkyg9e-0 ivadCR"
    id="ancestry-selected"
    onChange={[Function]}
    value="All"
  >
    <optgroup
      label="Ancestry"
    >
      <option
        value="All"
      >
        All Ancestries
      </option>
      <option
        value="African/African American"
      >
        African/African American
      </option>
      <option
        value="Amish"
      >
        Amish
      </option>
      <option
        value="Latino/Admixed American"
      >
        Latino/Admixed American
      </option>
      <option
        value="Ashkenazi Jewish"
      >
        Ashkenazi Jewish
      </option>
      <option
        value="East Asian"
      >
        East Asian
      </option>
      <option
        value="European (Finnish)"
      >
        European (Finnish)
      </option>
      <option
        value="Middle Eastern"
      >
        Middle Eastern
      </option>
      <option
        value="European (non-Finnish)"
      >
        European (non-Finnish)
      </option>
      <option
        value="South Asian"
      >
        South Asian
      </option>
      <option
        value="Other"
      >
        Other
      </option>
    </optgroup>
  </select>,
  <table
    className="Table__BaseTable-sc-7fgtt2-0 how-many-samples-are-in-each-mtdna-haplogroup-for-each-nuclear-ancestry-population__HaplogroupAncestryTable-dzc7cx-0 eMztGd"
  >
    <thead>
      <tr>
        <th
          scope="col"
        >
          Haplogroup
        </th>
        <th
          scope="col"
        >
          Population
        </th>
        <th
          scope="col"
        >
          Number of Samples
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <th
          scope="row"
        >
          A
        </th>
        <th>
          African/African American
        </th>
        <td>
          94
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          A
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          2380
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          A
        </th>
        <th>
          East Asian
        </th>
        <td>
          74
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          A
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          43
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          A
        </th>
        <th>
          Other
        </th>
        <td>
          84
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          A
        </th>
        <th>
          South Asian
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          B
        </th>
        <th>
          African/African American
        </th>
        <td>
          95
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          B
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          1178
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          B
        </th>
        <th>
          East Asian
        </th>
        <td>
          208
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          B
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          22
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          B
        </th>
        <th>
          Other
        </th>
        <td>
          32
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          B
        </th>
        <th>
          South Asian
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          C
        </th>
        <th>
          African/African American
        </th>
        <td>
          53
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          C
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          687
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          C
        </th>
        <th>
          East Asian
        </th>
        <td>
          47
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          C
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          37
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          C
        </th>
        <th>
          Other
        </th>
        <td>
          39
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          C
        </th>
        <th>
          South Asian
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          African/African American
        </th>
        <td>
          37
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          182
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          East Asian
        </th>
        <td>
          309
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          12
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          29
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          Other
        </th>
        <td>
          12
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          D
        </th>
        <th>
          South Asian
        </th>
        <td>
          22
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          E
        </th>
        <th>
          African/African American
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          E
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          E
        </th>
        <th>
          East Asian
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          E
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          4
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          E
        </th>
        <th>
          Other
        </th>
        <td>
          6
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          F
        </th>
        <th>
          African/African American
        </th>
        <td>
          21
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          F
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          3
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          F
        </th>
        <th>
          East Asian
        </th>
        <td>
          229
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          F
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          6
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          F
        </th>
        <th>
          South Asian
        </th>
        <td>
          23
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          G
        </th>
        <th>
          African/African American
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          G
        </th>
        <th>
          East Asian
        </th>
        <td>
          60
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          G
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          G
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          17
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          G
        </th>
        <th>
          South Asian
        </th>
        <td>
          8
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          African/African American
        </th>
        <td>
          306
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          Amish
        </th>
        <td>
          164
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          329
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          357
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          East Asian
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          2198
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          11226
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          Other
        </th>
        <td>
          152
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          H
        </th>
        <th>
          South Asian
        </th>
        <td>
          46
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          African/African American
        </th>
        <td>
          9
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          10
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          68
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          18
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          546
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          Other
        </th>
        <td>
          6
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          HV
        </th>
        <th>
          South Asian
        </th>
        <td>
          42
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          African/African American
        </th>
        <td>
          25
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          30
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          East Asian
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          114
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          743
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          Other
        </th>
        <td>
          8
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          I
        </th>
        <th>
          South Asian
        </th>
        <td>
          8
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          African/African American
        </th>
        <td>
          61
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          141
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          107
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          245
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          3
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          2520
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          Other
        </th>
        <td>
          41
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          J
        </th>
        <th>
          South Asian
        </th>
        <td>
          26
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          African/African American
        </th>
        <td>
          52
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          30
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          460
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          194
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          1952
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          Other
        </th>
        <td>
          33
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          K
        </th>
        <th>
          South Asian
        </th>
        <td>
          10
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L0
        </th>
        <th>
          African/African American
        </th>
        <td>
          605
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L0
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          35
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L0
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          10
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L0
        </th>
        <th>
          Other
        </th>
        <td>
          13
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L1
        </th>
        <th>
          African/African American
        </th>
        <td>
          2760
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L1
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          149
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L1
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L1
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          28
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L1
        </th>
        <th>
          Other
        </th>
        <td>
          38
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L1
        </th>
        <th>
          South Asian
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          African/African American
        </th>
        <td>
          4404
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          160
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          34
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          East Asian
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          44
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          Other
        </th>
        <td>
          78
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L2
        </th>
        <th>
          South Asian
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          African/African American
        </th>
        <td>
          5282
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          242
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          East Asian
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          39
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          Other
        </th>
        <td>
          104
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L3
        </th>
        <th>
          South Asian
        </th>
        <td>
          3
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L4
        </th>
        <th>
          African/African American
        </th>
        <td>
          118
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L4
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          4
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L4
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L4
        </th>
        <th>
          Other
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          L5
        </th>
        <th>
          Other
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          African/African American
        </th>
        <td>
          113
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          3
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          10
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          East Asian
        </th>
        <td>
          341
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          64
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          Other
        </th>
        <td>
          3
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          M
        </th>
        <th>
          South Asian
        </th>
        <td>
          763
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          African/African American
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          122
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          East Asian
        </th>
        <td>
          52
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          7
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          161
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          Other
        </th>
        <td>
          9
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          N
        </th>
        <th>
          South Asian
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          P
        </th>
        <th>
          East Asian
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          P
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          African/African American
        </th>
        <td>
          5
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          4
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          28
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          East Asian
        </th>
        <td>
          103
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          29
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          2
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          69
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          Other
        </th>
        <td>
          4
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          R
        </th>
        <th>
          South Asian
        </th>
        <td>
          149
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          African/African American
        </th>
        <td>
          66
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          Amish
        </th>
        <td>
          13
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          34
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          63
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          239
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          2599
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          Other
        </th>
        <td>
          34
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          T
        </th>
        <th>
          South Asian
        </th>
        <td>
          32
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          African/African American
        </th>
        <td>
          185
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          Amish
        </th>
        <td>
          190
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          101
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          56
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          1238
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          3
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          3892
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          Other
        </th>
        <td>
          90
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          U
        </th>
        <th>
          South Asian
        </th>
        <td>
          282
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          V
        </th>
        <th>
          African/African American
        </th>
        <td>
          16
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          V
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          14
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          V
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          47
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          V
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          273
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          V
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          869
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          V
        </th>
        <th>
          Other
        </th>
        <td>
          15
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          African/African American
        </th>
        <td>
          15
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          6
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          23
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          214
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          508
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          Other
        </th>
        <td>
          14
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          W
        </th>
        <th>
          South Asian
        </th>
        <td>
          38
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          African/African American
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          Amish
        </th>
        <td>
          25
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          Latino/Admixed American
        </th>
        <td>
          18
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          Ashkenazi Jewish
        </th>
        <td>
          9
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          59
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          Middle Eastern
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          405
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          Other
        </th>
        <td>
          7
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          X
        </th>
        <th>
          South Asian
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Y
        </th>
        <th>
          East Asian
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Y
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Z
        </th>
        <th>
          East Asian
        </th>
        <td>
          28
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Z
        </th>
        <th>
          European (Finnish)
        </th>
        <td>
          45
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Z
        </th>
        <th>
          European (non-Finnish)
        </th>
        <td>
          11
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Z
        </th>
        <th>
          Other
        </th>
        <td>
          1
        </td>
      </tr>
      <tr>
        <th
          scope="row"
        >
          Z
        </th>
        <th>
          South Asian
        </th>
        <td>
          4
        </td>
      </tr>
    </tbody>
    <tfoot>
      <tr>
        <th
          scope="row"
        >
          Haplogroup: 
          All
        </th>
        <th>
          Ancestry: 
          All
        </th>
        <td>
          Samples:
           
          56434
        </td>
      </tr>
    </tfoot>
  </table>,
]
`;
